platform/ext/target/nordic_nrf/nrf5340pdk_nrf5340_cpuapp/README.rst

```diff
@@ -37,12 +37,9 @@ following commands:
 .. code:: bash
 
     $ mkdir build && cd build
-    $ cmake -G"Unix Makefiles" \
-            -DPROJ_CONFIG=`readlink -f ../configs/ConfigDefault.cmake` \
-            -DTARGET_PLATFORM=nrf5340pdk_nrf5340_cpuapp \
-            -DCMAKE_BUILD_TYPE=Debug \
-            -DBL2=False \
-            -DCOMPILER=GNUARM \
+    $ mkdir build && cd build
```

Comment:
Now these sample outputs are similar to the NXP board.

Reply:
But why is that line present twice?

Reply:
I'll fix it in my next PR

Reply:
ups sorry @anangl  this came from copy-paste.
ashamed now